source/making_payments/index.html.md.erb

```diff
@@ -25,6 +25,13 @@ The call to create a payment with the GOV.UK Pay API is:
 
 `POST  /v1/payments`
 
+You must send the following parameters in your request body when creating a payment:
+
+* the amount the user will pay, in pence (`amount`)
+* a reference to help you identify the payment (`reference`)
+* a human-readable description of the purpose of the payment (`description`)
+* a URL GOV.UK Pay will redirect your user to after th user completes their payment (`return_url`)
```

Comment:
Could you, please review this line? t's a bit, hmm, funny

Reply:
I've edited this now.